packages/react/react-query/test-d/index.test-d.ts

```diff
@@ -0,0 +1,13 @@
+import { expectType } from 'tsd';
+import { SuspendedUseQueryResultOnIdle, SuspendedUseQueryResultOnSuccess, useSuspendedQuery } from '../dist';
+
+const queryKey = ['example'] as const;
+const queryFn = async () => 'response';
+
+/* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
### Check it is fine please
if no eslint-disable comment, eslint emit error like below
<img width="972" alt="image" src="https://user-images.githubusercontent.com/61593290/199543124-abcc0611-93c4-419d-b149-854e70f93d37.png">